src/MassTransit/Retry.cs

```diff
@@ -68,23 +68,23 @@ public static IRetryPolicy Intervals(this IExceptionFilter filter, params TimeSp
         /// Create an interval retry policy with the specified intervals. The retry count equals
         /// the number of intervals provided
         /// </summary>
-        /// <param name="intervals">The intervals before each subsequent retry attempt</param>
```

Comment:
Can you remove this file from the commit? I'd prefer not to have this causing a build when it's mostly documentation.